CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [1.8.5] - 2023-11-22
+
+### Added
+
+- All credential types supported by Azure can now be used in `azure_msi` NodeAttestor plugin and `azure_key_vault` KeyManager plugin (#4568)
+- `EnableHostnameLabel` field in Server and Agent `telemetry` configuration section that enables addition of a hostname label to metrics (#4584)
+
+### Changed
+
+- Agent SDS API now provides a SPIFFEValidationContext as the default CertificateValidationContext (#4618)
```

Comment:
This was already the case for Envoy 1.18+, we just made it the default if we couldn't detect the Envoy version (inverted the check).... not sure how we want to word it, maybe:
```suggestion
- Agent SDS API now provides a SPIFFEValidationContext as the default CertificateValidationContext when the Envoy version cannot be determined (#4618)
```